specifyweb/frontend/js_src/lib/components/FormParse/cells.ts

```diff
@@ -332,9 +332,12 @@ export function parseFormCell(
       : cellType === 'Label'
       ? 'right'
       : 'left',
-    verticalAlign: f.includes(cellVerticalAlign, verticalAlign)
-      ? verticalAlign
-      : 'stretch',
+    verticalAlign:
+      cellType === 'SubView'
```

Comment:
This is wrong order of ternaries - you have to swap them.

First check if user explicitly specified a valid alignment, and if so use it, and only if they didn't specify an alignment, you should use a default, which would be determined by a ternary based on cellType

In your case, you prohibited the user from customizing the alignment for anything other than subview - defeats the point of the feature